cmd/skaffold/app/flags/filepath.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2019 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package flags
+
+import (
+	"os"
+)
+
+type filepathFlag struct {
+	path        string
+	shouldExist bool
+}
+
+func (f *filepathFlag) SetIfValid(value string) error {
+	copy := *f
```

Comment:
`copy` collides with a builtin. Maybe rename to `clone`?

Reply:
thanks! Will do